src/Central.php

```diff
@@ -458,10 +458,16 @@ private static function getMessages(): array
         $user = new User();
         $user->getFromDB(Session::getLoginUserID());
         if ($user->fields['authtype'] == Auth::DB_GLPI && $user->shouldChangePassword()) {
-            $expiration_msg = sprintf(
-                __('Your password will expire on %s.'),
-                Html::convDateTime(date('Y-m-d H:i:s', $user->getPasswordExpirationTime()))
-            );
+            $expire_time = $user->getPasswordExpirationTime();
```

Comment:
Maybe a method in `User` would be better? As far as I see, code is exactly the same in front page and here